tasks/RedHat.yml

```diff
@@ -1,4 +1,18 @@
 ---
+# Elasticsearch CentOS/RedHat install
+
+# Get elasticsearch version when elasticsearch is already installed
+- name: Get installed elasticsearch version
+  shell: rpm -qa --queryformat '%{version}' elasticsearch
+  args:
+    warn: false
+  register: installed_elasticsearch_version
+
+- name: Abort when elasticsearch installed version < 1.8 and version >=5 is going to be installed
+  fail:
+    msg: "Please uninstall elasticsearch {{ installed_elasticsearch_version.stdout }} before upgrading to {{ elasticsearch_version }} version"
+  when: installed_elasticsearch_version.stdout|trim != "" and installed_elasticsearch_version.stdout is version_compare('1.8', '<')  and elasticsearch_version is version_compare('5.0', '>=')
```

Comment:
I think that "and" is the default operator when you use more than one comparision, so writing it as:
```
when:
 - installed_elasticsearch_version.stdout|trim != "" 
 - installed_elasticsearch_version.stdout is version_compare('1.8', '<') 
 - elasticsearch_version is version_compare('5.0', '>=')
```

is easier to follow